config/configauth/README.md

```diff
@@ -1,10 +1,18 @@
-# Authentication configuration for receivers
+# Authentication configuration
 
-This module allows server types, such as gRPC and HTTP, to be configured to perform authentication for requests and/or RPCs. Each server type is responsible for getting the request/RPC metadata and passing down to the authenticator.
+This module defines necessary interfaces to implement 
 
-The currently known authenticators:
+- Server type authenticators for gRPC and HTTP receivers to perform authentication for requests and/or RPCs. Each server type is responsible for getting the request/RPC metadata and passing down to the authenticator.
+- Client type authenticators for gRPC and HTTP exporters to perform client side authentication. 
 
-- [oidc](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/oidcauthextension)
+Few such authenticators that are implemented currently:
```

Comment:
Those are not "a few authenticators", those are all known authenticators :-) Perhaps something like:

```suggestion
The currently known authenticators are:
```